src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  max-width: $card-width;
+
+   &__img {
+    display: block;
+
+    max-width: 160px;
+    width: 100%;
+    height: 134px;
+
+    margin: 0 auto;
+    margin-bottom: 40px;
+   }
+
+   &__title {
+    color: $main-color;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+   }
+
+   &__code {
+    color: $secondary-color;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    font-style: normal;
+    margin-bottom: 16px;
+   }
+
+   &__rating {
+    margin-bottom: 24px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 15px;
+    height: 16px;
```

Comment:
https://codeguide.academy/html-css.html#css-order

Check the same in other places